alea/model_configs/unbinned_wimp_statistical_model.yaml

```diff
@@ -45,24 +45,26 @@ parameter_definition:
     fit_guess: 1.0
     description: Parameter to account for the uncertain signal expectation given a certain cross-section
 
-  # er_band_shift:
-  #   nominal_value: 0
-  #   ptype: shape
-  #   uncertainty: scipy.stats.uniform(loc=-1, scale=2)
-  #   relative_uncertainty: false
-  #   fittable: true
-  #   blueice_anchors:
-  #     - -1
-  #     - 0
-  #     - 1
-  #   fit_limits:
-  #     - -1
-  #     - 1
-  #   description: ER band shape parameter (shifts the ER band up and down)
+  er_band_shift:
+    nominal_value: 0
+    ptype: shape
+    uncertainty: null  # scipy.stats.uniform(loc=-1, scale=2)
+    # relative_uncertainty: false
+    fittable: true
+    blueice_anchors:
+      - -2
+      - -1
+      - 0
+      - 1
+      - 2
+    fit_limits:
+      - -2
+      - 2
+    description: ER band shape parameter (shifts the ER band up and down)
 
 likelihood_config:
   likelihood_weights: [1, 1, 1]
-  template_folder: []
+  template_folder: alea/templates
```

Comment:
This one is not needed when we read a file from alea internal location. We can even delete this entire line.